<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>914</width>
         <height>427</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="currentFileName">
          <property name="text">
           <string>Current Processing File</string>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QTextEdit" name="textEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="saveButton">
            <property name="text">
             <string>Save</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveAsButton">
            <property name="text">
             <string>Save As</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="discardButton">
            <property name="text">
             <string>Discard</string>
            </property>
            <property name="shortcut">
             <string>Ctrl+D</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="2">
           <widget class="QLabel" name="endingRatio2Label">
            <property name="text">
             <string>Ending Ratio 2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QSpinBox" name="endingRatio2">
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QSpinBox" name="startingRatio3">
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="endingRatio1">
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="endingRatio1Label">
            <property name="text">
             <string>Ending Ratio 1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="startingRatio1Label">
            <property name="text">
             <string>Starting Ratio 1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="startingRatio1">
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QSpinBox" name="endingRatio3">
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="startingRatio2Label">
            <property name="text">
             <string>Starting Ratio 2</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QSpinBox" name="startingRatio2">
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="startingratio3Label">
            <property name="text">
             <string>Starting Ratio 3</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="endingRatio3Label">
            <property name="text">
             <string>Ending Ratio 3</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="origGCodeLabel">
            <property name="text">
             <string>Original GCode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="origGCodeComboBox">
            <property name="minimumSize">
             <size>
              <width>700</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="origGCodeLocation">
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="mixedGCodeLabel">
            <property name="text">
             <string>   Mixed Gcode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="mixedGCodeComboBox">
            <property name="minimumSize">
             <size>
              <width>700</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="mixedGCodeLocation">
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="mixButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>MIX IT</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>934</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
